config/initializers/content_security_policy.rb

```diff
@@ -6,28 +6,50 @@
 # For further information see the following documentation
 # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy
 
-# Rails.application.configure do
-#   config.content_security_policy do |policy|
-#     policy.default_src :self, :https
-#     policy.font_src    :self, :https, :data
-#     policy.img_src     :self, :https, :data
-#     policy.object_src  :none
-#     policy.script_src  :self, :https
-#     policy.style_src   :self, :https
-#     # Specify URI for violation reports
-#     # policy.report_uri "/csp-violation-report-endpoint"
-#   end
-#
-#   # Generate session nonces for permitted importmap and inline scripts
-#   config.content_security_policy_nonce_generator = ->(request) { request.session.id.to_s }
-#   config.content_security_policy_nonce_directives = %w(script-src)
-#
-#   # If you are using webpack-dev-server then specify webpack-dev-server host
-#   if Rails.env.development?
-#     policy.connect_src :self, :https, "http://localhost:3035", "ws://localhost:3035"
-#   end
-#
-#   # Report CSP violations to a specified URI. See:
-#   # https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy-Report-Only
-#   # config.content_security_policy_report_only = true
-# end
+CONTENT_SECURITY_POLICY_MAP = {
+  base_uri: [:none],
+  form_action: [:self],
+  frame_ancestors: [:none],
+  connect_src: [:self],
+  default_src: [:self],
+  font_src: [:self],
+  img_src: [:self],
+  object_src: [:none],
+  script_src: [:self],
+  style_src: [:self]
+}.freeze
```

Comment:
What is the purpose of this layer of indirection?

Reply:
See https://app.asana.com/0/1142794766483633/1208315235207752

Reply:
Maybe it deserves a comment

Reply:
It makes it possible to add hosts to the policies more structured in commented groups, like the Rollbar addition on lines 23-25

Reply:
That makes total sense :+1: 

For some reason I did not notice that even though I looked through the code multiple times.